validation-test/Sema/type_checker_perf/fast/property_vs_unapplied_func.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: %target-typecheck-verify-swift -solver-expression-time-threshold=1 -solver-disable-shrink
+// REQUIRES: tools-release,no_asan
+
+struct Date {
+  var description: String
+  func description(with: Int) -> String { "" }
+}
+
+struct DatabaseLog {
+  let string: String
+  let values: [String]
+  let date: Date
+
+  var description: String {
+    return "[" + string + "] [" + date.description + "] " + string + " [" + values.joined(separator: ", ") + "]"
+  }
+}
+
+extension Sequence {
+  public func count(
+    where predicate: (Element) throws -> Bool
```

Comment:
If anybody knows of other `count(where:)` test cases, please let me know and I'll add them to this file!